vmdb/app/controllers/ems_common.rb

```diff
@@ -219,11 +219,11 @@ def form_field_changed
           # Hide/show C&U credentials tab
           page << "$('metrics_li').#{params[:server_emstype] == "rhevm" ? "show" : "hide"}();"
         end
-        if ["openstack"].include?(params[:server_emstype])
+        if ["openstack", "openstack_infra"].include?(params[:server_emstype])
```

Comment:
@martinpovolny Totally agree, but that seems like a job that's outside the scope of this PR.  I think it dalls into pluggable providers / UI for pluggable providers discussion.  cc @blomquisg